pkg/controller/servicemesh/webhooks/server.go

```diff
@@ -82,5 +86,13 @@ func Add(mgr manager.Manager) error {
 		Handler: validation.NewMemberValidator(),
 	})
 
+	log.Info("Adding readiness probe handler")
+	readinessProbeHandler := func(response http.ResponseWriter, request *http.Request) {
```

Comment:
Does this end up waiting until the webhook CAs are updated, or is that no longer necessary?